[Term Entry] Progress (#2842)
Codecademy/docs: content/html/concepts/semantic-html/terms/progress/progress.md

---

> Title: 'progress' 

Reviewer: ```suggestion
Title: '<progress>' 
```

Description: 'Used to display a progress bar.' 
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags: 
  - 'Semantic'
  - 'Tags'
CatalogContent: 
  - 'learn-html'
  - 'path/front-end--engineer-career-path'
---

The **`<progress>`** tag is a semantic HTML element that is used to a display a progress bar, an indicator that represents the progress a person has made in completing tasks such as a level in a game, question in an online exam, or parts of a [form](https://www.codecademy.com/resources/docs/html/forms).

> **Note:** When using the `max` attribute in the `<progress>` element, the value must be greater than `0` and be a valid floating point number(e.g., 5.3). If the value is not specified, the default value is `1`.

## Syntax

> ```html

Reviewer: Can we list the important attributes under syntax? Also show that there's inner text possible?

Author: Sure @SSwiniarski

<progress> </progress>
```

> ## Example 1

Reviewer: Can we have images showing the display of these examples?

---
Title: '<progress>' 
Description: 'Used to display a progress bar.' 
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags: 
  - 'Semantic'
  - 'Tags'
CatalogContent: 
  - 'learn-html'

>   - 'path/front-end--engineer-career-path'

Reviewer 2: ```suggestion
  - 'paths/front-end-engineer-career-path'
```

---

> The **`<progress>`** tag is a semantic HTML element that is used to a display a progress bar, an indicator that represents the progress a person has made in completing tasks such as a level in a game, question in an online exam, or parts of a [form](https://www.codecademy.com/resources/docs/html/forms).

Reviewer 2: ```suggestion
The **`<progress>`** tag is a semantic HTML element that is used to a display a progress bar, a graphic indicator that represents the status of an event or process such as the level in a game, question in an online exam, or parts of a [form](https://www.codecademy.com/resources/docs/html/forms).
```

## Syntax

```html
<progress value="number" max="number"> 
  <!-- Percentage of completed task goes here -->
  </progress>
```

> > **Note:** When using the `max` attribute in the `<progress>` element, the value must be greater than `0` and be a valid floating point number(e.g., 5.3). If the value is not specified, the default value is `1`.

Reviewer 2: ```suggestion
> **Note:** When using the `max` attribute in the `<progress>` element, the value must be greater than `0` and be a valid floating point number (e.g. 5.3). If the value is not specified, the default value is `1`.
```

> ## Example 1

Reviewer 2: I would recommend removing one of these examples because they're very similar. Additionally, it would be preferable to show an example that more closely resembles a real world application. It is unlikely that this tag would be used to display static values.

Author: Sure
